app/src/sharedTest/java/org/oppia/app/player/audio/CellularDataDialogFragmentTest.kt

```diff
@@ -0,0 +1,72 @@
+package org.oppia.app.player.audio
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import kotlinx.coroutines.CoroutineDispatcher
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.home.HomeActivity
+import org.oppia.util.threading.BackgroundDispatcher
+import org.oppia.util.threading.BlockingDispatcher
+import javax.inject.Singleton
+
+/** Tests for [CellularDataDialogFragment]. */
+@RunWith(AndroidJUnit4::class)
+class CellularDataDialogFragmentTest {
+
+  @Test
+  fun testCellularDataDialogFragment_loadDialogFragment_loadAudioFragment_loadLanguageFragment_isDisplayed() {
+    ActivityScenario.launch(HomeActivity::class.java).use {
+      Espresso.onView(withId(R.id.dummy_audio_button)).perform(click())
```

Comment:
Nit: prefer statically importing onView() for consistency with other static imports, and to reduce the code that needs to be read/written here.

Reply:
LGTM.